bms/templates/registration/header_sidebar.html

```diff
@@ -16,95 +16,102 @@
+                </form>
             </div>
+        </div>
+    </nav>
+</header>
+    <div id="wrapper" class="toggled">
+        <div id="sidebar-wrapper">
+        <ul class="sidebar-nav">
+            <li class="active">
+                <a href="/website_home"><i class="fa fa-fw fa-dashboard"></i> Dashboard</a>
+            </li>
 
-            <div id="page-content-wrapper">
-                <div class="container-fluid">
-                    {% block content %} {% endblock %}
-                </div>
+            <li>
+                <a href="javascript:;" data-toggle="collapse" data-target="#demo"><i class="fa fa-folder-open-o"></i> Project <i class="fa fa-fw fa-caret-right"></i></a>
+                <ul id="demo" class="collapse">
+                    <li>
+                        <a href="/create_project"><i class="fa fa-fw fa-plus-circle"></i>Create Project</a>
+                    </li>
+                    <li>
+                        <a href="/project_list"><i class="fa fa-fw fa-list"></i>Project List</a>
+                    </li>
+                </ul>
+            </li>
+            <li>
+                <a href="javascript:;" data-toggle="collapse" data-target="#demo1"><i class="fa fa-fw fa-bug"></i> Bug <i class="fa fa-fw fa-caret-right"></i></a>
+                <ul id="demo1" class="collapse">
+                    <li>
+                        <a href="/create_bug"><i class="fa fa-fw fa-plus-circle"></i>Create Bug</a>
+                    </li>
+                    <li>
+                        <a href="/bug_list"><i class="fa fa-fw fa-eye"></i>Bug View</a>
+                    </li>
+                </ul>
+            </li>
+        </ul>
+        </div>
+        
+        <div id="page-content-wrapper">
+            <div class="container-fluid">
+                {% block content %} {% endblock %}
             </div>
         </div>
-        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
-        <script type='text/javascript' src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
-        <script type="text/javascript" src="{% static 'bms_app/js/sidebar.js' %}"></script>
-        <script src="https://cdn.datatables.net/1.10.15/js/jquery.dataTables.min.js"></script>
-        <script src="https://cdn.datatables.net/1.10.15/js/dataTables.bootstrap.min.js"></script>
-        {% block jsfile %} {% endblock%}
-    </body>
+    </div>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
+    <script type='text/javascript' src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
+    <script type="text/javascript" src="{% static 'bms_app/js/sidebar.js' %}"></script>
+    <script src="https://cdn.datatables.net/1.10.15/js/jquery.dataTables.min.js"></script>
+    <script src="https://cdn.datatables.net/1.10.15/js/dataTables.bootstrap.min.js"></script>
+    {% block jsfile %} {% endblock%}
+</body>
```

Comment:
Why you have used <body> and <html> tags in this file it should be in base template and it should be extended in base template.